### PR TITLE
Convert DNS Seedlist spec to reStructuredText

### PR DESCRIPTION
**Pull request by derickr**

### CODE REVIEW

File: source/initial-dns-seedlist-discovery.rst [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+:Spec-ticket: SPEC-878
+:Title: Initial DNS Seedlist discovery
+:Authors: Derick Rethans
+:Status: Draft
+:Type: Standards
+:Last Modified: 2017-08-29
+:Version: 0.3
+:Spec Lead: Matt Broadstone
+:Advisory Group: A. Jesse Jiryu Davis
```
Comment (bjori) [inline source/initial-dns-seedlist-discovery.rst]: The `A` needs to be escaped, otherwise rst decides this is a A. B. C. D. list

Comment (jmikola) [reply source/initial-dns-seedlist-discovery.rst]: FYI: `\A.` should do the trick.

File: source/initial-dns-seedlist-discovery.rst [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+:Approver(s): Bernie Hackett (2017-08-17), David Golden (2017-08-29), Jeff Yemin (2017-08-22), Matt Broadstone (2017-08-24), A. Jesse Jiryu Davis (2017-08-14)
```
Comment (jmikola) [inline source/initial-dns-seedlist-discovery.rst]: Escape `\A.` here as well.

File: source/initial-dns-seedlist-discovery.rst [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+Initial DNS Seedlist discovery
+==============================
+
+:Spec-ticket: SPEC-878
+:Title: Initial DNS Seedlist discovery
+:Authors: Derick Rethans
+:Status: Draft
+:Type: Standards
+:Last Modified: 2017-08-29
+:Version: 0.3
```
Comment (jmikola) [inline source/initial-dns-seedlist-discovery.rst]: Shouldn't this start at 1.0 for a new spec?

File: source/initial-dns-seedlist-discovery.rst [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+Backwards Compatibility
+=======================
+
+There are no backwards compatibility concerns.
+
+Future Work
+===========
+
+In the future we could consider using the priority and weight fields of the
+SRV records, or to use SRV records to do mongos discovery.
```
Comment (jmikola) [inline source/initial-dns-seedlist-discovery.rst]: "MongoS" is used in the abstract. Should these be consistent?

File: source/initial-dns-seedlist-discovery.rst [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+:Title: Initial DNS Seedlist discovery
+:Authors: Derick Rethans
+:Status: Draft
+:Type: Standards
+:Last Modified: 2017-08-29
+:Version: 0.3
+:Spec Lead: Matt Broadstone
+:Advisory Group: A. Jesse Jiryu Davis
+:Approver(s): Bernie Hackett (2017-08-17), David Golden (2017-08-29), Jeff Yemin (2017-08-22), Matt Broadstone (2017-08-24), A. Jesse Jiryu Davis (2017-08-14)
+:Informed: drivers@
```
Comment (jmikola) [inline source/initial-dns-seedlist-discovery.rst]: This field shouldn't be necessary. I don't find it in the following recent specs:

 * https://github.com/mongodb/specifications/blob/master/source/extended-json.rst
 * https://github.com/mongodb/specifications/blob/master/source/change-streams.rst

File: source/initial-dns-seedlist-discovery.rst (hunk 3 above)
Comment (mbroadst) [reply source/initial-dns-seedlist-discovery.rst]: :+1:

File: source/initial-dns-seedlist-discovery.rst (hunk 2 above)
Comment (mbroadst) [reply source/initial-dns-seedlist-discovery.rst]: Interestingly enough, this case will not cause the list to happen. I originally solved the issue with Jesse's `A.` by simply moving him down the list of approvers in change streams

File: source/initial-dns-seedlist-discovery.rst (hunk 4 above)
Comment (mbroadst) [reply source/initial-dns-seedlist-discovery.rst]: I prefer it, `mongos` is very hard for me to skim visually

File: source/initial-dns-seedlist-discovery.rst (hunk 2 above)
Comment (jmikola) [reply source/initial-dns-seedlist-discovery.rst]: Ah, it's likely because his name is no longer the first term in the table cell.

**Comment by derickr** [comment]
Should have fixed all of these.

File: source/initial-dns-seedlist-discovery.rst [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+:Spec-ticket: SPEC-878
+:Title: Initial DNS Seedlist discovery
+:Authors: Derick Rethans
+:Status: Draft
+:Type: Standards
+:Last Modified: 2017-08-29
+:Version: 1.0
+:Spec Lead: Matt Broadstone
+:Advisory Group: \A. Jesse Jiryu Davis
+:Approver(s): Bernie Hackett (2017-08-17), David Golden (2017-08-29), Jeff Yemin (2017-08-22), Matt Broadstone (2017-08-24), A. Jesse Jiryu Davis (2017-08-14)
```
Comment (ajdavis) [inline source/initial-dns-seedlist-discovery.rst]: In the other specs we don't include the approver's dates.

**Comment by ajdavis** [comment]
I have just a nitpick. LGTM.

**Comment by derickr** [comment]
This was merged some time ago, closing this PR.

**Pull request closed without merging**